### PR TITLE
Document and test new keybindings option

### PR DESCRIPTION
**Pull request by souporserious**
Not sure if the README and CHANGELOG are where you want it. Feel free to change my wording. I added a test when all keybinding properties are in use. The `all arrows designated` test is already using an array so that should be ok right? Please let me know anything you want me to change.

### CODE REVIEW

**Comment by davidtheclark** [comment]
Awesome. Forgot to mention something I built-in that we need to document and test:

By default, we're assuming that undesignated modifier keys are `false`. So if you just specify `{ keyCode: 38 }`, that keybinding will be ignored if key 38 is combined with meta, ctrl, or alt.

File: README.md [hunk 1]
```diff
@@ -81,13 +81,16 @@ var myMegaMenuFocusGroup = createFocusGroup();
...
-**backArrows** { Array of `'up'`, `'down'`, `'left'`, or `'right'` }:
-Specify which arrows should move the focus *back* through the group (e.g. from index 4 to index 3).
-Default: `[ 'up' ]`
+Default:
+``` js
+{
+  next: { keyCode: 40 }, // ArrowDown
+  prev: { keyCode: 38 }, // ArrowUp
+}
+```
```
Comment (davidtheclark) [inline README.md:93]: Let's also provide examples using arrays and modifier keys.

**Comment by souporserious** [comment]
Added two more examples and a test to make sure holding modifier keys by default does not allow focus navigation 😃

File: README.md [hunk 2]
```diff
@@ -82,13 +82,29 @@ var myMegaMenuFocusGroup = createFocusGroup();
...
 **keybindings** { Object of `'next'`, `'prev'`, `'first'`, or `'last'` }:
-Specify which key events should move the focus *forward*, *back*, to the *first* member, or to the *last* member through the group. Provide objects (or arrays of objects) that describe the requirements of a [KeyboardEvent](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent) that should trigger that keybinding.
+Specify which key events should move the focus *forward*, *back*, to the *first* member, or to the *last* member through the group. Provide objects (or arrays of objects) that describe the requirements of a [KeyboardEvent](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent) that should trigger that keybinding. Undesignated modifier keys are false by default. So when using something like `{ keyCode: 38 }`, that keybinding will be ignored if key 38 is combined with meta, ctrl, or alt.
 
 Default:
 ``` js
 {
-  next: { keyCode: 40 }, // ArrowDown
-  prev: { keyCode: 38 }, // ArrowUp
+  next: { keyCode: 40 },
```
Comment (davidtheclark) [inline README.md]: Let's leave the comments there, for transparency.

File: README.md [hunk 3]
```diff
@@ -82,13 +82,29 @@ var myMegaMenuFocusGroup = createFocusGroup();
...
+
+Use arrays of objects for multiple key bindings:
+``` js
+{
+  next: [{ key: 'ArrowDown' }, { key: 'ArrowRight' }],
+  prev: [{ key: 'ArrowUp' }, { key: 'ArrowLeft' }],
+}
+```
+
+Even add modifier's or any valid event properties:
```
Comment (davidtheclark) [inline README.md]: No apostrophe needed for plural "modifiers".

**Comment by davidtheclark** [comment]
Great! Two picky comments then we'll merge and release.

File: README.md (hunk 3 above)
Comment (souporserious) [inline README.md]: Oops! Haha good catch.

**Comment by souporserious** [comment]
Awesome! I force pushed over with the requested changes so it should be good now.

**Pull request merged**